alerts/redis/fragmentation-ratio.json

```diff
@@ -0,0 +1,36 @@
+{
+    "displayName": "Redis - Memory Fragmentation Ratio < 1",
+    "documentation": {
+        "content": "A fragmentation ratio less than 1.0 means that Redis is under-provisioned and requires more memory than is available on the system and so it has resorted to using swap memory resources thus reducing overall performance.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "Redis - Memory Fragmentation Ratio < 1",
+            "conditionThreshold": {
+                "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/redis.memory.fragmentation_ratio\"",
+                "aggregations": [
+                    {
+                        "alignmentPeriod": "300s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                ],
+                "comparison": "COMPARISON_LT",
+                "duration": "30s",
```

Comment:
condition_threshold.duration had an invalid value of \"30s\": only multiples of 1 minute are supported",